config/canonical.json

```
@@ -30336,8 +30339,11 @@
   },
   "shop/supermarket|Kiwi": {
     "count": 195,
+    "countryCodes":["no", "dk"],
     "tags": {
       "brand": "Kiwi",
+      "brand:wikidata": "Q1613639",
+      "brand:wikipedia": "en:Kiwi_(store)",
```

Can you remove the `_` - thanks!
```suggestion
      "brand:wikipedia": "en:Kiwi (store)",
```